.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,11 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Consul-K8s Community Support
```

Comment:
Do we want to still direct people to discuss?

Reply:
I'd like to set up a template for questions instead of sending people to discuss. My 2c.

Reply:
That sounds good, I had added a question template, and I'll go ahead and remove this link